components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/RoleManagementServiceImpl.java

```diff
@@ -863,6 +863,17 @@ public List<String> getRoleIdListOfUser(String userId, String tenantDomain) thro
             }
         }
         List<String> roles = roleDAO.getRoleIdListOfUser(userId, tenantDomain);
+        try {
+            if (!OrganizationManagementUtil.isOrganization(tenantDomain)) {
```

Comment:
What if we have a method to add everyone role ID to a given list of roles.
We can keep this service level method clean by invoking that method. The business logic can be moved to that private method.